pytorch_toolkit/image_classification/model_templates/custom-classification/mobilenet_v3_small/main_model.yaml

```diff
@@ -0,0 +1,99 @@
+lr_finder:
+  enable: True
+  mode: brute_force
+  stop_after: False
+  num_epochs: 4
+  num_iter: 8
+  epochs_warmup: 1
+  path_to_savefig: 'lr_finder.jpg'
```

Comment:
where it is saved exactly? Temp dir? Current dir? Working dir?

Reply:
Users can specify any directory. In this case, it's just for example. If left as is it will be saved in the working directory with this name.